### ISSUE TITLE
PDU Board added to electronics hangs below the frame - can hit extruder in highest Z position

### ISSUE THREAD

**Issue by bryn51** [davidkerny/BUZA-ULTIMULTI#2]
Yesterday, my Upgrade kit froMMU3 arrived, and the only thing I needed for it, having built UltiMulti weeks ago, was PDU Board.
So I installed it, then went through some routine tests.  Extruder at some stage went to highest Z position, and I discovered if extruder was moved left/right, it strikes the PDU Board. Also some wires dangling beneath the MMU hit the extruder wire harness. Can't be good.  Can't ignore it.
I considered mounting electronics above the board, but this would involve throwing away some of the Ulti Multi benefits  so its a no to that idea. 
So, now I am looking at the legs design, both standard and Bear, to see if MMU can be mounted higher by about 5mm. will post a remake on printables if it succeeds.

**Comment by bryn51** [comment]
I made a remix on Printables to alter the legs and accommodate the PDU without interference to Extruder movement at highest Z position.  https://www.printables.com/model/626377-mmu3ultimulti-revised-leg-for-pdu

You are welcome to include that alteration in this distribution if you approve of it, and you wish to do that.

**Comment by davidkerny** [comment]
Hi! 
The idea for the Ultimulti C version (electronics on the bottom) was to bend the PD board addon to the back so that it doesnt take up as much space below the MMU unit. 

These is a small notch on the inner side of the right-side leg indicating how much to bend the PD board's connector to make it fit. After bending, the PD-board should snap into the notch. The PD-board in this position might be visible in some of the photos on Printables.
Bending the connector might be creepy. Some users will probably be uncomfortable with doing it and might find your design better. So, thanks for the remix!

**Comment by bryn51** [comment]
Hi David, and thanks for responding,.

As background, I have a life experience of 20 years working as electronics technician for a large Telco, and this included board repair to a high standard. (Repair of damage caused by lightning strikes).

But now I’m a “little” older and retired from all that.  So, with that experience and knowledge of consequences should I F&*k it up, and not having spare boards on hand, I am indeed reticent about bending the connector pins in that way.  If the pins should break or become unsoldered then it will be unreliable, and any damage like fine cracks is not always visible.
Since the goal of PDU board is to make the MMU3 more reliable than MMU2, bending the pins seems to work against that theme.

I have made the replacement legs that were remixed, and the modification seems to work OK.  Time will tell if I made them strong enough.  I used the strongest material I have on hand, but only 2 perimeters and 15% infill.

My suggestion would be to add a section to the printables and github readme about that modification, as it is not obvious, but I had noted the little slot on the original left side leg and wondered about its purpose.  It just did not occur to me to bend the pins. (Duh – right !).
Inclusion of the two photographs would be a bonus.

Anyway, I am happy for you to make the altered leg design available since it has now been tested and seems to work.

Thanks for a great design on the UltiMulti, it must have taken a LOT of effort, which is commendable.

I have the MMU3 for Mk4 printer on order.  As a full kit it comes with everything already printed. Easy to just go with it.  But the utility of the UltiMulti is just so much better, I am wondering now if I should prepare for “someday soon” release of the MK4 version of MMU by printing the UltiMulti plastics over again.

Anyway, Tks again and Cheers,
Bryn Parrott



From: davidkerny ***@***.***>
Date: Friday, October 27, 2023 at 09:21
To: davidkerny/BUZA-ULTIMULTI ***@***.***>
Cc: Bryn Parrott ***@***.***>, State change ***@***.***>
Subject: Re: [davidkerny/BUZA-ULTIMULTI] PDU Board added to electronics hangs below the frame - can hit extruder in highest Z position (Issue #2)

Hi!
The idea for the Ultimulti C version (electronics on the bottom) was to bend the PD board addon to the back so that it doesnt take up as much space below the MMU unit.

These is a small notch on the inner side of the right-side leg indicating how much to bend the PD board's connector to make it fit. After bending, the PD-board should snap into the notch. The PD-board in this position might be visible in some of the photos on Printables.
Bending the connector might be creepy. Some users will probably be uncomfortable with doing it and might find your design better. So, thanks for the remix!
